ur_controllers/src/scaled_joint_trajectory_controller.cpp

```diff
@@ -55,24 +55,28 @@ controller_interface::return_type ScaledJointTrajectoryController::update()
   }
 
   if (get_current_state().id() == lifecycle_msgs::msg::State::PRIMARY_STATE_INACTIVE) {
-    if (!is_halted) {
-      halt();
-      is_halted = true;
-    }
     return controller_interface::return_type::OK;
   }
 
-  auto resize_joint_trajectory_point = [](trajectory_msgs::msg::JointTrajectoryPoint& point, size_t size) {
+  auto resize_joint_trajectory_point = [&](trajectory_msgs::msg::JointTrajectoryPoint& point, size_t size) {
     point.positions.resize(size);
-    point.velocities.resize(size);
-    point.accelerations.resize(size);
+if (has_velocity_state_interface_) {
```

Comment:
Indentation seems to be broken here.
```suggestion
    if (has_velocity_state_interface_) {
```